src/main/java/seedu/address/logic/commands/RemindCommand.java

```diff
@@ -7,14 +7,19 @@
 import seedu.address.model.person.ReminderDatePredicate;
 
 /**
- * Finds and lists all persons in CakeCollate database whose delivery date is within 3 days of the current date.
+ * Finds and lists all orders in CakeCollate database whose delivery date is within 3 days of the current date.
```

Comment:
If you're changing such that user can choose the range of dates then you should remove "is within 3 days of the current date" right.

Reply:
changed